featuretools/entityset/entity.py

```diff
@@ -413,7 +413,17 @@ def delete_variables(self, variable_ids):
         """
         Remove variables from entity's dataframe and from
         self.variables
+
+        Args:
+            variable_ids (list): Variables to delete
```

Comment:
Let's be just a little more specific and indicate the list should contain strings:
`variable_ids (list[str]): List of variable to delete`

Reply:
yes, you are correct